README.md

```diff
@@ -61,15 +61,18 @@ documentation. The website should be consulted for further information:
 If you use mlpack in your research or software, please cite mlpack using the
 citation below (given in BiBTeX format):
 
-    @article{mlpack2013,
-      title     = {{mlpack}: A Scalable {C++} Machine Learning Library},
-      author    = {Curtin, Ryan R. and Cline, James R. and Slagle, Neil P. and
-                   March, William B. and Ram, P. and Mehta, Nishant A. and Gray,
-                   Alexander G.},
-      journal   = {Journal of Machine Learning Research},
-      volume    = {14},
-      pages     = {801--805},
-      year      = {2013}
+    @article{mlpack2018,
```

Comment:
Nice!

Reply:
Yeah, I forgot to do this earlier, sorry about that. :)